addons/Dimmer/Dimmer.lua

```diff
@@ -104,7 +119,7 @@ windower.register_event('addon command',function(...)
         windower.chat.input('//dimmer')
```

Comment:
I know this was here before your edit, but I'm not liking how it inputs the command again instead of just doing `search_item()`. Would be cool if you could change that.